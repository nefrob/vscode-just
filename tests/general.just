# Comments

# I am a comment
