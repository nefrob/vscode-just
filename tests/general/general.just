# Comments

# Standard comment

something # Comment after code

# Numbers

123
# TODO: should the "." be colored differently?
123.456

# Imports

import 'foo/bar.just'

mod bar
