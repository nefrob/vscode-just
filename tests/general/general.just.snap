># Comments
#^^^^^^^^^^ source.just comment.line.number-sign.just
>
># Standard comment
#^^^^^^^^^^^^^^^^^^ source.just comment.line.number-sign.just
>
>something # Comment after code
#^^^^^^^^^^ source.just
#          ^^^^^^^^^^^^^^^^^^^^ source.just comment.line.number-sign.just
>
># Imports
#^^^^^^^^^ source.just comment.line.number-sign.just
>
>import 'foo/bar.just'
#^^^^^^ source.just keyword.other.reserved.just
#      ^ source.just
#       ^ source.just string.quoted.single.just string.quoted.single.just
#        ^^^^^^^^^^^^ source.just string.quoted.single.just
#                    ^ source.just string.quoted.single.just
>import? 'foo/bar.just'
#^^^^^^ source.just keyword.other.reserved.just
#      ^ source.just keyword.operator.optional.just
#       ^ source.just
#        ^ source.just string.quoted.single.just string.quoted.single.just
#         ^^^^^^^^^^^^ source.just string.quoted.single.just
#                     ^ source.just string.quoted.single.just
>
>mod bar
#^^^ source.just keyword.other.reserved.just
#   ^ source.just
#    ^^^ source.just variable.name.module.just
>mod? bar
#^^^ source.just keyword.other.reserved.just
#   ^ source.just keyword.operator.optional.just
#    ^ source.just
#     ^^^ source.just variable.name.module.just
>mode bar 'foo/bar.just'
#^^^^^^^^^ source.just
#         ^ source.just string.quoted.single.just string.quoted.single.just
#          ^^^^^^^^^^^^ source.just string.quoted.single.just
#                      ^ source.just string.quoted.single.just
>